lms/static/sass/features/_learner-profile.scss

```diff
@@ -1,10 +1,94 @@
 // lms - application - learner profile
 // ====================
 
-// Table of Contents
-// * +Container - Learner Profile
-// * +Main - Header
-// * +Settings Section
+.learner-achievements {
+    .learner-message {
+        @extend %no-content;
```

Comment:
@andy-armstrong: Most css changes I don't feel confident reviewing.  I'm wondering:
1. Whether @HarryRein should do a quick pass, and
2. What css work/training I should do to be better equipped for this.

Reply:
I think it is a good idea for @HarryRein to review the CSS in this PR. I don't think it makes sense for you to take CSS training at the moment, but you could ask Mike what he thinks.